pyproject.toml

```
@@ -109,4 +110,4 @@ disable = [
     "too-many-statements",
     "unspecified-encoding",
     "wildcard-import"
-]
+]
```

Do we need to remove this newline at the end of the file?
```suggestion
]

```

---

I am not removing the new empty line at the end of the file; I'm adding it.

---

My apologies @thiagogenez, my mistake.